libcodechecker/analyze/analyzer.py

```diff
@@ -159,14 +159,20 @@ def perform_analysis(args, context, actions, metadata):
 
     actions_map = create_actions_map(actions, manager)
 
+    ctu_reanalyze_on_failure = False
```

Comment:
Why would it return `None`? `args.ctu_reanalyze_on_failure` can only be `True` if the user specifies the opt-string. In case the optstring is not specified, accessing via `.` throws a `KeyError`, and `get`ing it would return a `None`, but if the key is *not* in the `Namespace` (because the user didn't specify the opt-string), then the `in` is `False` and `and` is short-circuiting.